### PR TITLE
chore: use astyle from the lvgl organization

### PR DESCRIPTION
**Pull request by kisvegabor**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Comment by szepeviktor** [comment]
I've just released all 3.4.x versions.
https://github.com/szepeviktor/astyle

**Comment by szepeviktor** [comment]
Here is my secret `.sf/fix.sh`
```bash
# find . -not -path "./.sf/*" -not -path "./.git/*" -type f|xargs -t -I% -- .sf/fix.sh "%"
chmod -x "$1"
dos2unix -q -k "$1"
git add "$1"
```

**Comment by lhdjply** [comment]
> I've just released all 3.4.x versions. https://github.com/szepeviktor/astyle

Can you update to 3.5? I am using 3.5

**Comment by kisvegabor** [comment]
I can't see why the action is still failing. Now both should use the same version of astyle.

> Can you update to 3.5? I am using 3.5

Is it worth updating to v3.5?

**Comment by szepeviktor** [comment]
> @szepeviktor Can you update to 3.5? I am using 3.5

As you wish! It is done.
Please transfer the usual 一百万元 🙃

**Comment by lhdjply** [comment]
> I can't see why the action is still failing. Now both should use the same version of astyle.
  
Are you sure install 3.4.12

Screenshot_20240801_220121_com huawei browser

> Is it worth updating to v3.5?

I don't know. I just prefer new stuff

> Please transfer the usual 一百万元 🙃

一百五元？

File: scripts/code-format.cfg [hunk 1]
```diff
@@ -25,18 +25,27 @@
...
---exclude=../src/extra/libs/qrcode/qrcodegen.c
---exclude=../src/extra/libs/qrcode/qrcodegen.h
---exclude=../src/extra/libs/sjpg/tjpgd.c
---exclude=../src/extra/libs/sjpg/tjpgd.h
---exclude=../src/extra/libs/sjpg/tjpgdcnf.h
---exclude=../src/misc/lv_style_gen.c
---exclude=../src/misc/lv_style_gen.h
---exclude=../src/lv_conf_internal.h
---exclude=../tests/src/test_cases/_test_template.c
+--exclude=../src/libs/gif/gifdec.c
```
Comment (lhdjply) [inline scripts/code-format.cfg:33]: The path was wrong. You write the path is v9, but this is v8

**Comment by kisvegabor** [comment]
I don't understand what's happening. The reported formatting issue is [not there](https://github.com/kisvegabor/lvgl_upstream/blob/fix/formatting/src/extra/layouts/flex/lv_flex.c#L285). There are no extra spaced in this line.

**Comment by lhdjply** [comment]
> I don't understand what's happening. The reported formatting issue is [not there](https://github.com/kisvegabor/lvgl_upstream/blob/fix/formatting/src/extra/layouts/flex/lv_flex.c#L292-L293). There are no extra spaced in this line.
> 
> Any idea would be very welcome.

Please first rebase onto v8.4, and then run code-format; that should resolve the issue.

File: scripts/code-format.cfg (hunk 1 above)
Comment (lhdjply) [reply scripts/code-format.cfg:33]: please revert this

**Comment by kisvegabor** [comment]
> Please first rebase onto v8.4, and then run code-format; that should resolve the issue.

It was up to date. If it work on your end, could you send a PR based on this one?

**Comment by lhdjply** [comment]
Screenshot_20240817_232614_com huawei browser

Yours is really not based on the latest, you are missing the above commit.

Because the CI of the PR is based on the check after the merge, an error is reported. And you can look at the CI in your own warehouse,In your own warehouse check CI will not report error, because your own warehouse CI is not based on the merged.
So please rebase, you can solve it

**Comment by kisvegabor** [comment]
Ah, you are right... Thank you!  I was pulling `release/v8.4` from my fork and from the `lvgl` organization. 

Rebasing failed with many conflicts so I opened a new PR instead. #6694

**Pull request closed without merging**